model/lora-device-address.cc

```diff
@@ -179,12 +179,12 @@ LoraDeviceAddress::GetType(void)
 }
 
 uint32_t
-LoraDeviceAddress::Get(void) const
+LoraDeviceAddress::Get() const
 {
     NS_LOG_FUNCTION_NOARGS();
 
     uint32_t address = 0;
-    uint32_t nwkId = uint32_t(m_nwkId.Get() << 25);
+    auto nwkId = uint32_t(m_nwkId.Get() << 25);
```

Comment:
Would 
`uint32_t nwkId = m_nwkId.Get() << 25`
achieve the same result? If so, I suggest this